aws/policy/application-services.yaml

```diff
@@ -102,105 +121,44 @@ Statement:
       - cloudformation:CreateStack
       - cloudformation:DeleteChangeSet
       - cloudformation:DeleteStack
-      - cloudformation:DescribeChangeSet
-      - cloudformation:DescribeStackEvents
-      - cloudformation:DescribeStacks
-      - cloudformation:GetStackPolicy
-      - cloudformation:GetTemplate
-      - cloudformation:List*
       - cloudformation:SetStackPolicy
       - cloudformation:UpdateStack
       - cloudformation:UpdateTerminationProtection
       - cloudwatch:DeleteAlarms
-      - cloudwatch:DescribeAlarms
       - cloudwatch:PutMetricAlarm
-      - codebuild:CreateProject
```

Comment:
They are nor resource restricted right now. I moved these into resource unrestricted, but if we want to restrict those, I can add the restrictions (same for codecommit, sns, states).

Reply:
Can we keep all the actions that create, modify or delete resources in this sid? They aren't really resource restricted, but they are region restricted. I'm not sure why it's done this way, but it may be that some of these don't support the `RequestedRegion` condition. If they do, we could move these to a different region restricted sid and remove the individual resource restrictions.

Reply:
Done! I hope I haven't forgotten anything!